addons/website_event_questions/views/event_registration_views.xml

```diff
@@ -24,4 +24,15 @@
             </sheet>
         </field>
     </record>
+
+    <record id="view_event_registration_tree" model="ir.ui.view">

```

Comment:
New views should be correctly xml-id-ed, so "event_registration_view_tree" :) (even if base view is still using an old formatting)